### ISSUE TITLE
TypeError when testing using HttpClient login

### ISSUE THREAD

**Issue by LeonKoole** [django/channels#475]
Running Channels 1.0.1 on Django 1.8.17 with Python version 2.7.9
When testing, login runs into a TypeError. [This line](https://github.com/django/channels/blob/master/channels/tests/http.py#L130) expects `'FakeRequest'` to be a string, but it's unicode. This issue seems to have been caused by the commit 8a93dfc 2 days ago, where unicode_literals was imported into tests/http.py.

See this traceback:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/usr/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/usr/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/usr/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/usr/lib/python2.7/site-packages/django/test/testcases.py", line 962, in setUpClass
    cls.setUpTestData()
  File "/home/docker/hellolily/lily/calls/api/tests.py", line 18, in setUpTestData
    cls.channels_client.login(username=cls.user_obj.email, password='password')
  File "/usr/lib/python2.7/site-packages/channels/tests/http.py", line 115, in login
    self._login(user)
  File "/usr/lib/python2.7/site-packages/channels/tests/http.py", line 130, in _login
    request = type('FakeRequest', (object, ), {'session': self.session, 'META': {}})
TypeError: type() argument 1 must be string, not unicode
```

This issue can be reproduced with these 3 lines from the [documentation](https://channels.readthedocs.io/en/stable/testing.html#clients):
```
client = HttpClient()
user = User.objects.create_user(username='test', email='test@test.com',
                                password='123456') # fuck you security
client.login(username='test', password='123456')
```

**Comment by Krukov** [comment]
God damn, blame Unicode! :rage1:

**Comment by andrewgodwin** [comment]
Fixed by #476.
